Grand Rapids
Init, weather = Overcast
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Overcast
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Overcast
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly Cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Clear
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Clear
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/venv/lib/python3.11/site-packages/urllib3/connection.py", line 198, in _new_conn
    sock = connection.create_connection(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/shriya/weather-wallpaper/venv/lib/python3.11/site-packages/urllib3/util/connection.py", line 60, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/lib/python3.11/socket.py", line 962, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
socket.gaierror: [Errno 8] nodename nor servname provided, or not known

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/venv/lib/python3.11/site-packages/urllib3/connectionpool.py", line 787, in urlopen
    response = self._make_request(
               ^^^^^^^^^^^^^^^^^^^
  File "/Users/shriya/weather-wallpaper/venv/lib/python3.11/site-packages/urllib3/connectionpool.py", line 493, in _make_request
    conn.request(
  File "/Users/shriya/weather-wallpaper/venv/lib/python3.11/site-packages/urllib3/connection.py", line 445, in request
    self.endheaders()
  File "/opt/anaconda3/lib/python3.11/http/client.py", line 1289, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/opt/anaconda3/lib/python3.11/http/client.py", line 1048, in _send_output
    self.send(msg)
  File "/opt/anaconda3/lib/python3.11/http/client.py", line 986, in send
    self.connect()
  File "/Users/shriya/weather-wallpaper/venv/lib/python3.11/site-packages/urllib3/connection.py", line 276, in connect
    self.sock = self._new_conn()
                ^^^^^^^^^^^^^^^^
  File "/Users/shriya/weather-wallpaper/venv/lib/python3.11/site-packages/urllib3/connection.py", line 205, in _new_conn
    raise NameResolutionError(self.host, self, e) from e
urllib3.exceptions.NameResolutionError: <urllib3.connection.HTTPConnection object at 0x1091637d0>: Failed to resolve 'ip-api.com' ([Errno 8] nodename nor servname provided, or not known)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/venv/lib/python3.11/site-packages/requests/adapters.py", line 667, in send
    resp = conn.urlopen(
           ^^^^^^^^^^^^^
  File "/Users/shriya/weather-wallpaper/venv/lib/python3.11/site-packages/urllib3/connectionpool.py", line 841, in urlopen
    retries = retries.increment(
              ^^^^^^^^^^^^^^^^^^
  File "/Users/shriya/weather-wallpaper/venv/lib/python3.11/site-packages/urllib3/util/retry.py", line 519, in increment
    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='ip-api.com', port=80): Max retries exceeded with url: /json/?fields=status,message,country,countryCode,regionName,city,timezone,query (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x1091637d0>: Failed to resolve 'ip-api.com' ([Errno 8] nodename nor servname provided, or not known)"))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 8, in <module>
    print(loc.get_city())
          ^^^^^^^^^^^^^^
  File "/Users/shriya/weather-wallpaper/location.py", line 20, in get_city
    return self.location_getter()["city"]
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/shriya/weather-wallpaper/location.py", line 13, in location_getter
    r = requests.get(url = self.url)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/shriya/weather-wallpaper/venv/lib/python3.11/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/shriya/weather-wallpaper/venv/lib/python3.11/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/shriya/weather-wallpaper/venv/lib/python3.11/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/shriya/weather-wallpaper/venv/lib/python3.11/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/shriya/weather-wallpaper/venv/lib/python3.11/site-packages/requests/adapters.py", line 700, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='ip-api.com', port=80): Max retries exceeded with url: /json/?fields=status,message,country,countryCode,regionName,city,timezone,query (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x1091637d0>: Failed to resolve 'ip-api.com' ([Errno 8] nodename nor servname provided, or not known)"))
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly Cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Overcast
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Moderate or heavy rain with thunder
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Light rain
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Light rain
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Heavy rain
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Overcast
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Overcast
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Overcast
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Clear
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Overcast
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Overcast
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Overcast
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Overcast
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Overcast
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Clear
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Clear
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Patchy light rain with thunder
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Overcast
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Mist
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Overcast
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Light rain
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Light rain
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Overcast
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Overcast
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Patchy light rain with thunder
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly Cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly Cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly Cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Sunny
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
Grand Rapids
Init, weather = Partly cloudy
Traceback (most recent call last):
  File "/Users/shriya/weather-wallpaper/main.py", line 11, in <module>
    wallpaper.capture_wallpaper()
  File "/Users/shriya/weather-wallpaper/wallpaper.py", line 27, in capture_wallpaper
    with open(config_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shriya/assets/config.json'
